### DISCUSSION TITLE
Welcome to JsSIP Discussions!

### DISCUSSION THREAD

**Discussion by ibc** [versatica/JsSIP#854] [Announcements]
## 👋 Welcome!

We’re using GitHub Discussions as a forum for users to ask questions about JsSIP and help others.

📢 **Announce to your community** 

The former JsSIP public mailing list has been closed. Use GitHub Discussions instead.

**Comment by jyrkisaario** [comment]


**Reply by ibc** [reply]
Please open a new topic instead of replying to the forum announcement.

**Reply by jyrkisaario** [reply]
Ok, thanks
